pkg/operation/botanist/kubecontrollermanager_test.go

```diff
@@ -201,12 +201,28 @@ var _ = Describe("KubeControllerManager", func() {
 
 				It("hibernation enabled", func() {
 					botanist.Shoot.HibernationEnabled = true
-
+					kubernetesClient.EXPECT().Client().Return(c)
+					c.EXPECT().Get(ctx, kutil.Key(namespace, "kube-controller-manager"), gomock.AssignableToTypeOf(&appsv1.Deployment{})).DoAndReturn(func(_ context.Context, _ types.NamespacedName, obj *appsv1.Deployment) error {
```

Comment:
Can you add another test where the `kube-controller-deployment` is not found? This should be the default case if you create a new shoot with activated hibernation, right?

Reply:
Good suggestion, thanks. I'll add it in a sec.